src/components/ImportRecordsFlow/stepTwo/__test__/preview.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Preview from '../preview';
+import StyledFileCardTitle from '../styled/fileCardTitle';
+import StyledFileCardDescription from '../styled/fileCardDescription';
+import StyledButtonIcon from '../styled/buttonIcon';
+
+describe('<Preview />', () => {
+    it('should show default labels', () => {
+        const component = mount(<Preview />);
+        const title = component.find(StyledFileCardTitle);
+        const type = component.find(StyledFileCardDescription);
+        expect(title.text()).toBe('Unknow File Name');
+        expect(type.text()).toBe('Unknow File Type');
+    });
+    it('should set the right labels according to params', () => {
+        const params = { fileName: 'preview-title', fileType: 'text/csv' };
+        const component = mount(<Preview {...params} />);
+        const title = component.find(StyledFileCardTitle);
+        const type = component.find(StyledFileCardDescription);
+        expect(title.text()).toBe('preview-title');
+        expect(type.text()).toBe('text/csv');
+    });
+    it('should run remove handler function after click the remove button', () => {
+        const onRemoveFileFn = jest.fn();
+        const component = mount(<Preview onRemoveFile={onRemoveFileFn} />);
+        component.find(StyledButtonIcon).simulate('click');
+        expect(onRemoveFileFn).toHaveBeenCalled();
+    });
+    it('should have data passed properly to preview table', () => {
+        const props = {
+            data: [
+                { First_Name: 'John', Last_Name: 'Doe' },
+                { First_Name: 'Marie', Last_Name: 'Sun' },
+                { First_Name: 'Jane', Last_Name: 'Doe' },
+            ],
+            columns: ['First_Name', 'Last_Name'],
+            isLoading: true,
+        };
+        const component = mount(<Preview {...props} />);
+        const expectedPassedData = {
+            data: component.find('PreviewTable').prop('data'),
+            columns: component.find('PreviewTable').prop('columns'),
+            isLoading: component.find('PreviewTable').prop('isLoading'),
+        };
+        expect(expectedPassedData).toEqual(props);
+    });
+});
```

Comment:
I think we should put PreviewTable in other file and test it:
- should render a spinner when isLoading is passed
- should render a table when isLoading is not passed
- should render the right amount of columns